nilearn/regions/region_extractor.py

```diff
@@ -389,6 +419,23 @@ class RegionExtractor(NiftiMapsMasker):
         List of separated regions with each region lying on an
         original volume concatenated into a 4D image.
 
+    maps_img_ : :obj:`nibabel.nifti1.Nifti1Image`
+        The maps mask of the data.
+
+    %(nifti_mask_img_)s
+
+    n_elements_ : :obj:`int`
+        The number of overlapping maps in the mask.
+        This is equivalent to the number of volumes in the mask image.
+
+        .. versionadded:: 0.9.2
+
+    allow_overlap : True
+        If False, an error is raised if the maps overlaps
+        (ie at least two maps have a non-zero value for the same voxel).
```

Comment:
this is the default value from the parent class, but shouldn't we expect region extraction to extract non overlapping regions (from reading the doc string of the class)?

https://nilearn.github.io/dev/modules/generated/nilearn.regions.RegionExtractor.html#nilearn.regions.RegionExtractor